src/util/route.js

```diff
@@ -21,6 +21,8 @@ export function createRoute (
   const route: Route = {
     name: location.name || (record && record.name),
     meta: (record && record.meta) || {},
+    children: (record && record.children) || [],
+    parent: (record && record.parent) || {},
```

Comment:
Isn't it better to have `null` or `undefined` if the parent does not existL

Reply:
Yes, I think it would be. I'll update it.